content/03.results.md

```diff
@@ -168,11 +167,11 @@ We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medu
 By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
 Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (**Figure {@fig:S2}H**).
 The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (**Figure {@fig:S3}B-C**).
-We identified chromothripsis events in 41% (N = 19/46) of non-midline high-grade gliomas (high-grade glioma astrocytomas) and in 28.2% (N = 11/39) of diffuse midline gliomas (**Figure {@fig:Fig3}D**).
-We also found evidence of chromothripsis in over 15% of sarcomas, metastatic secondary tumors, embryonal tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, and ependymomas, highlighting the genomic instability and complexity of these pediatric brain tumors.
+We identified chromothripsis events in 28% (N = 11/39) of diffuse midline gliomas and 40% (N = 19/48) of non-midline HGGs (other HGGs) (**Figure {@fig:Fig3}D**).
```

Comment:
I would reverse this parenthetical so the "primary" matches the figure - 
```suggestion
We identified chromothripsis events in 28% (N = 11/39) of diffuse midline gliomas and in 40% (N = 19/48) of other HGGs (non-midline HGGs) (**Figure {@fig:Fig3}D**).
```

That said, I'm not convinced we need to actually say "non-midline HGGs." Having two terms floating around for the same cancer group might get confusing. The right approach may become clearer in final rounds of copy editing.

Reply:
Ah, yeah I suppose we should define other as non-midline somewhere because for this specific histology, that is something important clinicians etc want to know.

Reply:
Oops, reading and responding on my phone, I did not see that non-midline was retained. Ha.